installation-usage.md

```diff
@@ -81,6 +81,36 @@ spec:
       image: nginx
 ```
 
+You can find the profile path of the seccomp profile by checking the
+`seccompProfile.localhostProfile` attribute:
+
+```sh
+$ kubectl --namespace my-namespace get seccompprofile profile1 --output wide
+NAME       STATUS   AGE   SECCOMPPROFILE.LOCALHOSTPROFILE
+profile1   Active   14s   operator/my-namespace/custom-profiles/profile1.json
```

Comment:
I really like that, that will improve the usability a lot!

Reply:
That looks very neat, really good stuff @cmurphy. :+1: